qa/rpc-tests/test_framework/mininode.py

```diff
@@ -46,6 +46,11 @@
 
 COIN = 100000000 # 1 btc in satoshis
 
+NODE_NETWORK = (1 << 0)
+NODE_GETUTXO = (1 << 1)
+NODE_BLOOM = (1 << 2)
+NODE_WITNESS = (1 << 3)
```

Comment:
I thought Witness was a way to include more txs in blocks so it's surprising me that this needs to be a service bit that would influence node selection among nodes. Where can I read more about Witness to understand why this is needed please?